eng/scripts/Publish-MavenPackages.ps1

```diff
@@ -133,18 +133,37 @@ foreach ($packageDetail in $packageDetails) {
   Write-Information "Staging Description Option is: $stagingDescriptionOption"
 
   if ($RepositoryUrl -match "https://pkgs.dev.azure.com/azure-sdk/\b(internal|public)\b/*") {
+    if(Test-ReleasedPackage -RepositoryUrl $RepositoryUrl -PackageDetail $packageDetail -BearerToken $RepositoryPassword)
+    {
+      Write-Information "Package $($packageDetail.FullyQualifiedName) already deployed. Skipping deployment."
```

Comment:
This one might be worth doing Write-Host given that it is important information even if we don't have full logging enabled.

Reply:
If we use Write-Host, I can add that.  I'm wary as it's not pipeable.

Reply:
We have used Write-Host in a few places where we always wanted the the output but for our actual usages of this script in DevOps we generally include all the output anyway so in the end it probably doesn't matter much.

Reply:
Also for better or worse we don't end up writing most of our scripts where they are pipeable and we often times try to avoid piping in our scripts because of the error handling issues we have chatted about. So it isn't super important to worry much the pipeline because we have a lot of work to do if we really want that supported correctly.

Reply:
BTW I think this might be the only script we use Write-Information. We have used Write-Host, Write-Debug and Write-Verbose in a number of scripts but I think this is the only one that uses Write-Information. We might want to think a little more about the pros and cons of which logging types we use.

Reply:
I'll remove this in a following refactor PR